.github/workflows/continuous-integration.yml

```diff
@@ -15,7 +15,6 @@ jobs:
     runs-on: "ubuntu-20.04"
     env:
       SYMFONY_REQUIRE: ${{matrix.symfony-require}}
-      SYMFONY_DEPRECATIONS_HELPER: ${{matrix.symfony-deprecations-helper}}
```

Comment:
I think reason why we didn't simply put it to phpunit.xml but here only was that even if we don't want CI build to fail, we want tests to fail locally at least, so that people writing the PR are aware they might be introducing a new deprecation. I would say we should stick to that.